core/config/strategyLib.js

```diff
@@ -0,0 +1,31 @@
+import { importManager } from '../../more/imports/importManager';
+
+import { natStrategy } from './strategies/natStrategy';
+import { makeUniStrategy } from './strategies/uniStrategy';
+import { makePixelStrategy } from '../../more/pixels/pixelStrategy';
+
+import { insist } from '../../util/insist';
+import { mustBeComparable } from '../../util/sameStructure';
+
+const makeCustomInsistKind = (
+  descriptionCoercer = () => true,
+) => optDescription => {
+  insist(
```

Comment:
The purpose of customInsistKind seems to be to allow these checks around `descriptionCoercer` to be configurable. I doubt this is a good idea. I think these extra checks should be universal and only the remaining `descriptionCoercer` behavior should be configurable.